61_UI/include/nbl/this_example/renderpass/scene.hpp

```diff
@@ -0,0 +1,1362 @@
+#ifndef __NBL_THIS_EXAMPLE_SCENE_H_INCLUDED__
+#define __NBL_THIS_EXAMPLE_SCENE_H_INCLUDED__
+
+#include <nabla.h>
+
+#include "nbl/asset/utils/CGeometryCreator.h"
+#include "SBasicViewParameters.hlsl"
+#include "CAssetConverter.h"
+#include "geometry/creator/spirv/builtin/CArchive.h"
+#include "geometry/creator/spirv/builtin/builtinResources.h"
+
+enum E_OBJECT_TYPE : uint8_t
+{
+	EOT_CUBE,
+	EOT_SPHERE,
+	EOT_CYLINDER,
+	EOT_RECTANGLE,
+	EOT_DISK,
+	EOT_ARROW,
+	EOT_CONE,
+	EOT_ICOSPHERE,
+
+	EOT_COUNT,
+	EOT_UNKNOWN = std::numeric_limits<uint8_t>::max()
+};
+
+struct OBJECT_META
+{
+	E_OBJECT_TYPE type = EOT_UNKNOWN;
+	std::string_view name = "Unknown";
+};
```

Comment:
probably need some namespace around this

Reply:
bump

Reply:
resolved